Cirumflex_Combined.ino

```diff
@@ -1,12 +1,19 @@
 //Fixed colors set as globals. 03-Pattern fades between them.
```

Comment:
**this box** is not part of the code. It's a comment _about_ this set of changes. i.e. a "conversation".

What does this page mean? it's a `diff`.

The column on the left is the OLD file. On the right is the new file.

in this case, the header above tells you that I also renamed the file. Git will do the versioning, instead of us naming it crazy.

The only thing shown on this page are things that changed. Each change has some lines before and after to give context.

If I deleted a line, it will show as `-` and pink on the left. On the right, it shows what I substituted for it: `+` and green for replace, or just a lot of whitespace if I deleted.

Reply:
This conversation boxes explain why I made a change (as opposed to _what_ a line of code is for, which would be a comment in the code).

You are explicitly invited to `reply` to one of these boxes. And, to add a box to discuss some piece of code: find the line(s) that you want to discuss, hover just to the right of the line number (magic `+` appears), click the `+`, type discussion point in the box, click "add single comment".

Reply:
You will probably click `start a review`, which is almost the same thing. `add single comment` immediately submits your text. `start a review` remembers it till you click `finish your review` (up in the top-right).

**Don't forget to click `finish your review` if this happens, or I'll never see your comments**

Reply:
I'm going to do one of these "diffs" for each step of changing the code, hopefully in bite-sized chunks.